GA-worksheet.md

```diff
@@ -26,21 +26,21 @@
 ##Project #2 Part #2: Style the Header with CSS
 
 1. What advantages does the `<link>` tag have over the `<style>` tag?
-
+The link tag is a link to something or a heading. It also makes a specific style.
```

Comment:
Both `<link>` and `<style>` can be used to add style to a page, but link `<link>` isn't specific to style.

* `<link>` defines a link between a document and an external file. This means we can add style to a website by including a `<link>` tag to the separate CSS file
* `<style>` lets you add style directly to the HTML document. 

Think about why one would prefer `<link>`.